humanoid_league_team_communication/config/team_communication_config.yaml

```diff
@@ -1,28 +1,31 @@
-# The port to be used
-port: 12321
+# Only used if local_mode is false
+target_host: 127.0.0.1
+target_port: 3737
+receive_port: 3737
+
+# Only used when target_host is 127.0.0.1 (local mode)
+# the team communication will bind to one of these ports and send to the other ports, depending on its bot_id
+local_target_ports:
+  - 4001
+  - 4002
+  - 4003
+  - 4004
 
 # Rate of published messages in Hz
 rate: 1
 
 # average walking speed in [m/s]
-avg_walking_speed: 0.01
-
-# set the color of your team
-# 2: red
-# 3: blue
-# other values are invalid
-team_color: 2
-
-#lifetime of data [s]
-lifetime: 3
+avg_walking_speed: 0.2
 
-#threshold for necessary belief in data to be send to teammates [0,1]
-belief_threshold: 0
+# lifetime of data [s]
+lifetime: 1
 
 # ROS Topics
-team_data: 'team_data'
-strategy: 'strategy'
-robot_state: 'robot_state'
-position: 'pose_with_certainty'
-ball: 'balls_relative'
-obstacles: 'obstacles_relative'
+team_data_topic: 'team_data'
+strategy_topic: 'strategy'
+pose_topic: 'pose_with_certainty'
+ball_topic: 'balls_relative'
```

Comment:
Maybe we want to use the ball from the ball filter here

Reply:
I think that should wait until we use the ball filter in the behavior because current problems with the ball filter, most of all the problematic velocity detection, are also problematic here

Reply:
While I agree with you I also need to say, that the ball filter is currently used for the ball obstacle. We don't use it for the behavior yet due to same issues with the velocity, but I should have clear advantages that cancel out the disadvantages in this context, such as the confidence estimation based on the time since the last observation and it works in the map frame. The current implementation with ball relative only publishes the last observation, so if a robot knows the ball position but doesn't sees it right now it's not considered. The ball filter also provides constant updates and reduces the amount of explicit handling for multiple balls or no balls at all.